nightwatch-vrt.d.ts

```diff
@@ -0,0 +1,47 @@
+interface NightwatchVRTSettings {
```

Comment:
We'd prefer to only use JavaScript on this project. Could you remove the typescript files please?

Reply:
Would your team be willing to keep the definition file? It doesn't have to be any extra work to maintain. It is likely Typescript users will update the `.d.ts` if in the future you change the API . Otherwise I will have to add the definition to [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) - a less pleasant and time consuming experience.